File: Source/AlphaTab/Model/AccidentalType.cs
namespace AlphaTab.Model
{
    /// <summary>
    /// Defines all possible accidentals for notes.
    /// </summary>
    public enum AccidentalType
    {
        /// <summary>
        /// No accidental
        /// </summary>
        None,
        /// <summary>
        /// Naturalize 
        /// </summary>
        Natural,
        /// <summary>
        /// Sharp
        /// </summary>
        Sharp,
        /// <summary>
        /// Flat
        /// </summary>
        Flat,
        /// <summary>
        /// Natural for smear bends
        /// </summary>
        NaturalQuarterNoteUp,
        /// <summary>
        /// Sharp for smear bends
        /// </summary>
        SharpQuarterNoteUp,
        /// <summary>
        /// Flat for smear bends
        /// </summary>
        FlatQuarterNoteUp
    }
}

File: src/model/AccidentalType.ts
/**
 * Defines all possible accidentals for notes.
 */
export enum AccidentalType {
    /**
     * No accidental
     */
    None = 0,
    /**
     * Naturalize
     */
    Natural = 1,
    /**
     * Sharp
     */
    Sharp = 2,
    /**
     * Flat
     */
    Flat = 3,
    /**
     * Natural for smear bends
     */
    NaturalQuarterNoteUp = 4,
    /**
     * Sharp for smear bends
     */
    SharpQuarterNoteUp = 5,
    /**
     * Flat for smear bends
     */
    FlatQuarterNoteUp = 6,
    /**
     * Double Sharp, indicated by an 'x'
     */
    DoubleSharp = 7,
    /**
     * Double Flat, indicated by 'bb'
     */
    DoubleFlat = 8
}
